lib/curl_setup.h

```diff
@@ -842,6 +842,24 @@
 #include "curl_setup_once.h"
 #endif
 
+#ifdef UNDER_CE
+#define getenv curl_getenv  /* Windows CE does not support getenv() */
+#define raise(s) ((void)(s))
+/* Terrible workarounds to make Windows CE compile */
+#define errno 0
+#define CURL_SETERRNO(x) ((void)(x))
+#define EAGAIN 11
+#define ENOMEM 12
+#define EACCES 13
+#define EEXIST 17
+#define EISDIR 21
+#define ENOSPC 28
+#define ERANGE 34
+#define strerror(x) "?"
+#else
+#define CURL_SETERRNO(x) (errno = (x))
```

Comment:
Rather than use CURL_SETERRNO everywhere we need to set errno, can't we just have a static int errno or a thread specific int or something for CE? The strerror macro and errno macro always 0 seems not great. Was this an issue with just CeGCC or was it an issue for Visual Studio as well?

Reply:
`errno` is missing as a concept from WinCE, regardless of toolchain.

Emulating it via a global var could work, but it's beyond my capacity,
and with no way to make any tests, it'd probably make things worse
as the simple stubs here. Which is certainly not great.

I think this could be a next step in making WinCE work, after sorting
out the builds. There are other mappings that are not great or
missing functionality (I tried to make this clear in the PR message).

Reply:
Ok